src/main/java/org/skriptlang/skript/bukkit/loottables/LootTableUtils.java

```diff
@@ -0,0 +1,87 @@
+package org.skriptlang.skript.bukkit.loottables;
+
+import org.bukkit.block.Block;
+import org.bukkit.block.BlockState;
+import org.bukkit.loot.LootTable;
+import org.bukkit.loot.Lootable;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Utility class for loot tables.
+ */
+public class LootTableUtils {
```

Comment:
I think most of these are unnecessary
all the methods that call one thing and then update state should be removed, since the user may want to do multiple things and should only update once at the end.